<launch>


  <include file="$(find realsense2_camera)/launch/rs_camera.launch"/> 
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager"/>

   <node name = "cup_detect" pkg = "cv_watch_cup" type="cup_detect" output = "screen"/>
   <node name="result" pkg="aruco_pose" type="CameraResult" output="screen"/>
     <node pkg="nodelet" clear_params="true" type="nodelet" name="aruco_detect" args="load aruco_pose/aruco_detect nodelet_manager">
        <remap from="image_raw" to="/camera/color/image_raw"/>
        <remap from="camera_info" to="/camera/color/camera_info"/>
        <param name="length" value="0.7"/>
    </node>
        <!-- <remap from="image_raw" to="/camera/color/image_raw"/> -->
        <!-- <remap from="camera_info" to="/camera/color/camera_info"/> -->
   <!-- </node>	 -->

   <node name="gui" pkg="eurobot_gui" type="demo_guiForCamera.py" output="screen"/>
   
    <param  name="XPose" type="int" value="350"/>
    <param  name="YPose" type="int" value="500"/>
    <param name = "cupDistance" type = "int" value = "120"/>
    <param name="circleSize" type="int" value="40"/>
</launch>
